Backend/src/main/java/com/example/backend/project/controller/ProjectController.java

```diff
@@ -42,18 +43,17 @@ public ResponseEntity getAllProjects(@RequestHeader String googleTokenEncoded){
 
     @PutMapping("/softLock/{projectId}")
```

Comment:
Ovo bi trebalo biti PatchMapping jer kad se azurira samo dio podatka to se obicno radi sa PATCH metodom.
_PUT is a method of modifying resource where the client sends data that updates the entire resource . PATCH is a method of modifying resources where the client sends partial data that is to be updated without modifying the entire data._

Reply:
The more you know.

Ali ozbiljno hvala, nikada nisam vidio to čudo, pa mi je drago da sam saznao da postoji.